apps/omg/test/omg/state/persistence_test.exs

```diff
@@ -174,6 +183,34 @@ defmodule OMG.State.PersistenceTest do
     state
   end
 
+  defp persist_standard_exitable_utxos(%Core{utxos: utxos} = state, addresses, db_pid) do
```

Comment:
oh wait, this is overdone. We're after "making sure that the logic in `Core` works with the results of the query".

just add a new test (or pick one exiting with some interesting utxos already in, but only if there's a sensible one to pick from) and in that test check whether

```
{:ok, utxos_query_result} = OMG.DB.utxos(db_pid)
Core.standard_exitable_utxos(utxos_query_result, address.addr)
```
works and makes sense (e.g. contains two utxos of alice or something), done.

The necessity to reach into the `%Core{utxos: utxos}` should be a test smell to you here, not to mention the necessity to maintain this test later on.